### PR TITLE
Adding argocd alerts in moc-infra

### PR DESCRIPTION
**Pull request by mubariskhan96**
Introducing alerts for argocd in moc-infra. Re: https://github.com/operate-first/apps/issues/1677

### CODE REVIEW

File: argocd/overlays/moc-infra/alerts.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            summary: "'{{ $labels.name }}' Application has sync status as '{{ $labels.sync_status }}'"
+        - alert: ArgoAppSyncFailed
+          expr: argocd_app_sync_total{phase!="Succeeded"} == 1  # Triggered when argocd-application is not succeeded
+          for: 1m
+          labels:
+            severity: warning
+          annotations:
+            summary: "'{{ $labels.name }}' Application has sync phase as '{{ $labels.phase }}'"
+        - alert: ArgoAppMissing
+          expr: absent(argocd_app_info)                         # Txriggered when argocd-application info is not found
```
Comment (dystewart) [inline argocd/overlays/moc-infra/alerts.yaml:24]: Triggered spelled incorrectly in comment

File: argocd/overlays/moc-infra/alerts.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: argocd-alerts
+spec:
+  groups:
+    - name: argocd-alerts
+      rules:
+        - alert: ArgoAppSyncFailed
+          expr: argocd_app_sync_total{phase!="Succeeded"} == 1  # Triggered when argocd-application is not succeeded
```
Comment (4n4nd) [inline argocd/overlays/moc-infra/alerts.yaml:10]: @mubariskhan96 can you add a description for this alert? I don't know how this metric works.

Comment (mubariskhan96) [reply argocd/overlays/moc-infra/alerts.yaml:10]: added a descripton!

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**